### PR TITLE
Fix headers and revert duplicate files for LBR related cmdlets

### PR DESCRIPTION
**Pull request by kelqi**

### CODE REVIEW

File: skype/skype-ps/skype/Get-CsTenantNetworkRegion.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+# Get-CsTenantNetworkRegion
+
+## SYNOPSIS
+
+## SYNTAX
+
+```
+Get-CsTenantNetworkRegion [-LocalStore] [[-Identity] <Object>] [-Tenant <Object>] [-Filter <Object>] [-AsJob]
+```
```
Comment (tomkau) [inline skype/skype-ps/skype/Get-CsTenantNetworkRegion.md]: There's an quirk with PlatyPS in that it sets all parameters to be Objects, when in fact some are strings or numerical.  I can't remember if this is fixed in a later version.  Ken would know.

Meanwhile, you should double check the cmdlet source code to see if the parameter types are correct, and update as needed.

File: skype/skype-ps/skype/Get-CsTenantNetworkRegion.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+## DESCRIPTION
+{{Fill in the Description}}
```
Comment (tomkau) [inline skype/skype-ps/skype/Get-CsTenantNetworkRegion.md]: You need to add descriptions, examples, etc.

Comment (kelqi) [reply skype/skype-ps/skype/Get-CsTenantNetworkRegion.md]: Engineers will add once it's checked in

File: skype/skype-ps/skype/Get-CsTenantNetworkRegion.md (hunk 1 above)
Comment (kelqi) [reply skype/skype-ps/skype/Get-CsTenantNetworkRegion.md]: New PR with fix. Need to add -Session param when running platyPS command to avoid this issue

**Comment by kelqi** [comment]
New PR with correct variable types in params.

**Pull request closed without merging**